docs/en-us/_sidebar.md

```diff
@@ -1,9 +1,16 @@
+Pyecharts is a class library for generating Echarts charts. Echarts is a data visualization JS library of Baidu Open Source. The graphics generated with Echarts are very visual, and pyecharts is designed to interface with Python, making it easy to use data generation diagrams directly in Python.
```

Comment:
please reference the description here:https://github.com/vinta/awesome-python/pull/1107